tests/Doctrine/Annotation/TokenTest.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Doctrine\Annotation;
+
+use Doctrine\Common\Annotations\DocLexer;
+use PhpCsFixer\Doctrine\Annotation\Token;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Doctrine\Annotation\Token
+ */
+final class TokenTest extends TestCase
+{
+    public function testDefaults()
+    {
+        $token = new Token();
+
+        $this->assertSame(DocLexer::T_NONE, $token->getType());
+        $this->assertSame('', $token->getContent());
+    }
+
+    public function testConstructorSetsValues()
+    {
+        $type = 42;
+        $content = 'questionable';
+
+        $token = new Token(
+            $type,
+            $content
+        );
+
+        $this->assertSame($type, $token->getType());
+        $this->assertSame($content, $token->getContent());
+    }
+
+    public function testCanModifyType()
+    {
+        $type = 42;
+
+        $token = new Token();
+
+        $token->setType($type);
+
+        $this->assertSame($type, $token->getType());
+    }
+
+    /**
+     * @dataProvider providerIsType
+     *
+     * @param mixed $type
```

Comment:
Please update with correct types (`int` and `int|int[]`) like for the tested method, and remove invalid test cases instead.